test/utils/findGitmojiCommand.spec.js

```diff
@@ -6,20 +6,38 @@ import * as stubs from './stubs'
 describe('findGitmojiCommand', () => {
   stubs.commands.map((command) => {
```

Comment:
I believe we can switch to [test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) to make this simpler ✨